assignments/arrays.js

```diff
@@ -65,19 +65,42 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
 console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
+let carThirtyThree = inventory[32];
```

Comment:
While your output does give you the intended answer, this is under the assumption that the owner knows the index of car with ID 33.  How would you solve the problem if you had not known the index of the element you need.